README.md

```diff
@@ -204,3 +204,26 @@ Wait for all the pods to have come up, before proceeding (this can take a few mi
 Log in to the web console for Openshift and click into the `localmcp` project, then select `Mobile`. Under `Mobile Enabled Services` select `fh-sync-server` and click `Integrations`. 
 Under `Mobile Service Integrations`, click on `Create Integration` next to `Keycloak`.
 
+### Developing APBs Locally
+
+Create a cluster pointing at your own docker organisation, this will also copy all the existing APBs in feedhenry to the dockerhub_org:
+```
+ansible-playbook playbook.yml -e "dockerhub_username=<dockerusername>" -e "dockerhub_password=<dockerpassword>" -e "dockerhub_org=<USE_THIS_VALUE>" --ask-become-pass
+```
+
+If the APB you wish to develop already exists in the feedhenry organisation, then syncing that APB can be skipped by adding:
+```
+-e "skip_apb=<apb-name>"
```

Comment:
nice - is that new ?

Reply:
yeah, I noticed a bunch of duplicates would show up if you were working on APB that was also cloned from feedhenry. It came in, in this PR: https://github.com/feedhenry/mcp-standalone/pull/142/files